test/sortedLastIndex/sortedLastIndex.test.js

```diff
@@ -5,7 +5,7 @@ test('Testing sortedLastIndex', (t) => {
   //For more information on all the methods supported by tape
   //Please go to https://github.com/substack/tape
   t.true(typeof sortedLastIndex === 'function', 'sortedLastIndex is a Function');
-  t.equal(sortedLastIndex([10, 20, 30, 30, 40], 30), 3, 'Returns the highest index to insert the element without messing up the list order');
+  t.equal(sortedLastIndex([10, 20, 30, 30, 40], 30), 4, 'Returns the highest index to insert the element without messing up the list order');
```

Comment:
The test is wrong now, as it should return `4`. This is due to the fact that inserting at index equal to `3` would make it so that the array would be `[10, 20, 30, 30 (the inserted entry), 30, 40]` while it should be `[10, 20, 30, (the inserted entry), 30, 30, 40]`.